modules/migration-downloading-logs.adoc

```diff
@@ -3,35 +3,33 @@
 // * migration/migrating_3_4/troubleshooting-3-4.adoc
 // * migration/migrating_4_1_4/troubleshooting-4-1-4.adoc
 // * migration/migrating_4_2_4/troubleshooting-4-2-4.adoc
+
 [id='migration-downloading-logs_{context}']
 = Downloading migration logs
 
-You can download the Velero, Restic, and Migration controller logs in the {mtc-short} web console to troubleshoot a failed migration.
+You can download the Velero, Restic, and `MigrationController` Pod logs in the {mtc-short} web console to troubleshoot a failed migration.
```

Comment:
You mentioned in your comment that you would check these. Can you please let me know if you are updating them to lowercase? Thanks.

Reply:
Can you clarify? I understood from the API guidelines that it's "`MigrationController` Pod" and "Verify that the pods are running". Are you saying that "`MigrationController` Pod" should be "`MigrationController` pod"? The wording of the instructions is not clear to me.

Reply:
Yes that is what I am wondering, since I don't think this sentence refers to an API pod object directly.

@bergerhoffer can you please clarify whether "Velero, Restic, and `MigrationController` Pod" should become "Velero, Restic, and `MigrationController` pod" or not? Thank you!

Reply:
Yes, "Pod" should be "pod" in this instance. Really, there should never be an instance where pod is capitalized (except at the beginning of the sentence), unless it is styled as `Pod` when directly referencing the object.

In this sentence though I would also question whether the original text of "Velero, Restic, and Migration controller" was correct, instead of putting those into monospace.

Reply:
Thank you Andrea, that is very helpful!

@apinnick based on this discussion, you could update this sentence with something like the following:

"You can download the Velero, Restic, and migration controller pod logs in the {mtc-short} web console to troubleshoot a failed migration."

Please also apply something similar to other similar references in the MTC book. Thanks!

Reply:
Thanks @pneedle-rh! The only part I'd question is migration controller. I think that we had discussions awhile back specifically for the Ingress Controller, and the determination was to capitalize it. So I think I've seen a lot of places capitalizing those specific controllers, but I'm not sure if we ever made it a rule for all controllers. I'll bring it up for discussion with the team.

Reply:
'Pod' changed to 'pod'

Reply:
Great, thank you!